graviti/portex/register.py

```diff
@@ -111,3 +113,30 @@ def __call__(self, portex_type: Type[_P]) -> Type[_P]:
             mapping[pyarrow_type_id] = portex_type
 
         return portex_type
+
+
+class RemoteFileTypeResgister:
+    """The class decorator to portex external type and the remote file type.
+
+    Arguments:
+        name: The portex external type name.
+
+    """
+
+    SCHEMA_TO_REMOTE_FILE: ClassVar[Dict[str, Type["RemoteFile"]]] = {}
+
+    def __init__(self, name: str) -> None:
```

Comment:
only name cannot used as the identifier of a Portex type, because the different types with same name is possible to appear in different repo or different commit.